stock_by_warehouse_sale/views/sale_order_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.stock.warehouse</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/form//field[@name='sequence']" position="after">
+                <div class="text-center alert alertdialog" groups="stock.group_stock_multi_warehouses">
```

Comment:
`role` not required here?

Reply:
In this case it is not required because it is an alertdialog and not an alert-info